### ISSUE TITLE
Weird Console Error Output in 1.20.6

### ISSUE THREAD

**Issue by Shiroiame-Kusu** [Dreeam-qwq/Leaf#52]
Latest Leaf 1.20.6 mojmap with no plugin enabled.
https://pastebin.com/ipxUBjJd
```
[09:30:40 ERROR]: Error sending packet clientbound/minecraft:disconnect (skippable? false) 
io.netty.handler.codec.EncoderException: Sending unknown packet 'clientbound/minecraft:disconnect' 
at net.minecraft.network.codec.IdDispatchCodec.encode(IdDispatchCodec.java:46) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.codec.IdDispatchCodec.encode(IdDispatchCodec.java:20) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.PacketEncoder.encode(PacketEncoder.java:26) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.PacketEncoder.encode(PacketEncoder.java:12) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) ~[netty-codec-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:893) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:875) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:984) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:868) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.ChannelOutboundHandlerAdapter.write(ChannelOutboundHandlerAdapter.java:113) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at net.minecraft.network.Connection$2.write(Connection.java:758) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:893) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:956) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:982) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:950) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:1000) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1025) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:306) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at net.minecraft.network.Connection.doSendPacket(Connection.java:504) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.Connection.sendPacket(Connection.java:486) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.Connection.send(Connection.java:448) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.Connection.send(Connection.java:434) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at net.minecraft.network.Connection.exceptionCaught(Connection.java:220) ~[leaf-1.20.6.jar:git-Leaf-"a46d325"] 
at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:346) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:325) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.fireExceptionCaught(AbstractChannelHandlerContext.java:317) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.handler.flush.FlushConsolidationHandler.exceptionCaught(FlushConsolidationHandler.java:159) ~[netty-handler-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:346) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:325) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.fireExceptionCaught(AbstractChannelHandlerContext.java:317) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.DefaultChannelPipeline$HeadContext.exceptionCaught(DefaultChannelPipeline.java:1377) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:346) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:325) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.DefaultChannelPipeline.fireExceptionCaught(DefaultChannelPipeline.java:907) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.handleReadException(AbstractNioByteChannel.java:125) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:177) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.109.Final.jar:4.1.109.Final] 
at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.109.Final.jar:4.1.109.Final] 
at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?] 
```

**Comment by liangcha385** [comment]
It looks like a packet error occurred when the receiving player disconnected
Can you explain what you were doing at that time?

**Comment by Shiroiame-Kusu** [comment]
> It looks like a packet error occurred when the receiving player disconnected Can you explain what you were doing at that time?

Nothing. Just start the server, and receive the error.
I've already upload the complete log above, just click the link.

**Comment by Dreeam-qwq** [comment]
Can't reproduce. If you see this error frequently, report to here again :/

**Comment by Dreeam-qwq** [comment]
The issue exists on upstream 
https://github.com/PaperMC/Paper/issues/10662

**Comment by Dreeam-qwq** [comment]
Fixed in e4d196b39e09891997621531aa4e6c021c6cb43e

Upstream (Paper) commit: https://github.com/PaperMC/Paper/commit/8e6554a80a1efefa48adb1f7733554e528fb45f5
